docs/docs-book/src/running-coffee-as-a-server.md

```diff
@@ -0,0 +1,84 @@
+# Coffee Server Documentation
+
+To run Coffee as a server and provide a JSON RPC interface, you can use the `coffee_httpd` binary.
+
+Please note that the server runs on `localhost` with port `8080`.
+
+## Starting the Coffee Server
+
+To start the Coffee server, run the following command:
+
+ ```shell
+ coffee_httpd --network <network>
+ ```
+
+Make sure the `coffee_httpd` binary is in your system PATH or in the current working directory.
+
+## Available HTTP Calls
+
+The Coffee server provides several HTTP endpoints to interact with the plugins and repositories.
+
+### Install a Plugin
+
+Use the following `curl` command to install a plugin:
+
+ ```shell
+ curl -X 'POST' \
+   'http://localhost:8080/install' \
+   -H 'accept: application/json' \
+   -H 'Content-Type: application/json' \
+   -d '{
+     "plugin": "summary"
+   }'
+ ```
+
+### Remove an Installed Plugin
+
+Use the following `curl` command to remove an installed plugin:
+
+ ```shell
+ curl -X 'POST' \
+   'http://localhost:8080/remove' \
+   -H 'accept: application/json' \
+   -H 'Content-Type: application/json' \
+   -d '{
+     "plugin": "summary"
+   }'
+ ```
+
+### List Installed Plugins
+
+Send a GET request to `http://localhost:8080/list` to list the installed plugins.
+
+### Add a Remote Repository
+
+Use the following `curl` command to add a remote repository:
+
+ ```shell
+ curl -X 'POST' \
+   'http://localhost:8080/remote/add' \
+   -H 'accept: application/json' \
+   -H 'Content-Type: application/json' \
+   -d '{
+     "repository_name": "folgore",
+     "repository_url": "https://github.com/coffee-tools/folgore"
+   }'
+ ```
+
+### Remove a Remote Repository
+
+Use the following `curl` command to remove a remote repository:
+
+ ```shell
+ curl -X 'POST' \
+   'http://localhost:8080/remote/rm' \
+   -H 'accept: application/json' \
+   -H 'Content-Type: application/json' \
+   -d '{
+     "repository_name": "folgore"
+   }'
+ ```
+
+ ### List Remote Repositories
+
```

Comment:
Remove all and keep just a single line `To see the available calls you can use the Swagger API available at http://localhost:8080`

Reply:
What do you think of  0d9e720 ?

Reply:
Much better but I leave some comments